### PR TITLE
Rename HasAccessChecksSuppressed to IgnoresAccessibility

### PR DESCRIPTION
**Pull request by gafter**

### CODE REVIEW

**Comment by gafter** [comment]
@jaredpar @AnthonyDGreen @OmerRaviv Please check this.

File: src/Compilers/CSharp/Portable/Compilation/SyntaxTreeSemanticModel.cs [hunk 1]
```diff
@@ -87,7 +87,7 @@ public override SyntaxTree SyntaxTree
         /// <summary>
         /// Returns true if this is a SemanticModel that ignores accessibility rules when answering semantic questions.
         /// </summary>
-        public override bool HasAccessChecksSuppressed
+        public override bool IgnoresAccessibility
         {
             get { return _hasAccessChecksSuppressed; }
```
Comment (OmerRaviv) [inline src/Compilers/CSharp/Portable/Compilation/SyntaxTreeSemanticModel.cs]: Rename field here as well

**Comment by gafter** [comment]
OK, renamed all the fields and parameters too.

**Comment by tmat** [comment]
:+1:

**Pull request merged**